api/src/main/resources/api/service_openapi.yaml

```diff
@@ -576,14 +583,128 @@ components:
           description: Repeated instances results of the search.
           nullable: true
           items:
-            $ref: '#/components/schemas/EntityInstanceStruct'
+            $ref: "#/components/schemas/EntityInstanceStruct"
         nextPageToken:
-          $ref: '#/components/schemas/NextPageToken'
+          $ref: "#/components/schemas/NextPageToken"
 
     EntityInstanceStruct:
       type: object
       description: A map of an entity instance attribute names to their value for this instance.
       additionalProperties:
         type: object
         nullable: true
-        $ref: '#/components/schemas/AttributeValue'
+        $ref: "#/components/schemas/AttributeValue"
+
+    UserData:
+      type: object
+      description: The root container for user generated data.
+      properties:
+        version:
+          type: integer
+          description: The version of the data to support forward and backward compatibility.
```

Comment:
nit: would probably clarify: "version of UserData", else this can be misconstrued as a reference to the underlay or API or dataset.

Reply:
Done.